appcenter-link-scripts/src/ios/PodFile.js

```diff
@@ -70,23 +69,17 @@ Podfile.prototype.install = function () {
     });
 };
 
-Podfile.searchForFile = function (cwd) {
-    const podFilePaths = glob.sync(path.join(cwd, 'Podfile'), { ignore: 'node_modules/**' });
-    if (podFilePaths.length > 1) {
-        debug(podFilePaths);
-        throw new Error('Found more than one Podfile in this project');
-    } else if (podFilePaths.length === 1) {
-        return podFilePaths[0];
-    } else {
-        debug(`No podfile found in ${cwd}`);
-        childProcess.execSync('pod init', { cwd });
+Podfile.initializePodfileIfNecessary = function (podfilePath) {
+    if (!fs.statSync(podfilePath).isFile()) {
```

Comment:
statSync throws an exception when file is not found, thus preventing from creating podfile